Tasks/src/StringArrayFilter.java

```diff
@@ -0,0 +1,96 @@
+import java.util.Scanner;
+
+public class StringArrayFilter {
+
+    public static void main(String[] args) {
+
+        final String Result = "Результат вашего запроса следующий :";
+        final String OneMore = "Желаете попробовать еще раз? \n1 | да \n2 | нет";
```

Comment:
uppercase it's ONE_MORE and so on. Please rewrite

Reply:
Done